scripts-available/CDB_Groups.sql

```diff
@@ -0,0 +1,124 @@
+-- Creates a new group
+CREATE OR REPLACE
+FUNCTION cartodb.CDB_Group_CreateGroup(group_name text)
```

Comment:
totally, we lack of that organization admin who has the rights to do all the stuff.